cypress/support/commands.js

```diff
@@ -39,6 +39,24 @@ Cypress.Commands.add('login', () => {
   cy.contains('Welcome back');
 });
 
+Cypress.Commands.add('toggleAll', () => {
+  cy.login();
+  cy.visit('/settings');
+  // cy.get('input[name="isLocked"]').click();
+  // cy.get('input[name="name"]').click();
+  cy.get('#privacy-settings')
+    .find('.toggle-not-active')
+    .each(element => {
+      return new Cypress.Promise(resolve => {
+        element.click();
+        setTimeout(() => {
+          resolve();
+        }, 300);
+      });
+    });
```

Comment:
Can't we click on each button and wait for it to become `.toggle-active`?

Reply:
This is related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/40764 . So, not reliably.

Reply:
Oh, that's still not fixed, ouch.  

If I understand this correctly, there _should_ be something that changes in the UI on `updateUserFlagComplete`, so you should be able to wait for that.